### ISSUE TITLE
Updating has broken the installation with service "stations" has neither an image nor a build context specified: invalid compose project

### ISSUE THREAD

**Issue by s7ntax** [AzuraCast/AzuraCast#5522]
### Installation Method

Docker Installation

### AzuraCast Release Channel

Stable Channel

### Current AzuraCast Version

latest

### What happened?

When running the update it failed and broke the installation. 

Ubuntu Server 20.04

```
service "stations" has neither an image nor a build context specified: invalid compose project
service "stations" has neither an image nor a build context specified: invalid compose project
service "stations" has neither an image nor a build context specified: invalid compose project
service "stations" has neither an image nor a build context specified: invalid compose project
```

**Comment by ItsMitchh** [comment]
Can you try running these commands? 
```
./docker.sh update-self
./docker.sh install-docker
./docker.sh install-docker-compose
```
Also if you have a Docker-compose Override file, you'll need to remove the `stations` prefix as per this update
https://github.com/AzuraCast/AzuraCast/issues/5191

**Comment by s7ntax** [comment]
Thank you. It seems I had to edit the override file. 

However now it has broken the TLS and seems to throw up errors now on all browsers.

**Comment by s7ntax** [comment]
Yes it seems the update has broken the tls certificate. I run Azuracast on custom ports as I also run Apache which used 80/443 and even stopping Apache and changing ports in Azuracast to 80/443 it seems to fail to update the tls certificate. 

Can I just roll back to a previous version as I had it working fine until I tried to update it. I had a work around for tls by stopping Apache, changing ports in Azuracast to 80/443 and then changing back to custom once the tls certificate was updated.
